### PR TITLE
Ear 1368 answer properties prototype 2

### PR DESCRIPTION
**Pull request by martyncolmer**
Prototype for answer properties

### CODE REVIEW

File: eq-author/src/components/AnswerContent/AnswerProperties/DurationProperties.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            return (
+              <optgroup label={durationType} key={durationType}>
+                {map(unit, (durationConfig, key) => {
+                  const optionLabel = durationConfig.abbreviation;
+                  return (
+                    <option value={`${durationConfig.duration}`} key={key}>
+                      {optionLabel}
+                    </option>
+                  );
+                })}
```
Comment (MattElcock) [inline eq-author/src/components/AnswerContent/AnswerProperties/DurationProperties.js:50]:
```suggestion
                {map(unit, (durationConfig, key) => (
                    <option value={`${durationConfig.duration}`} key={key}>
                      {durationConfig.abbreviation}
                    </option>
                  );
                )}
```

File: eq-author/src/components/AnswerContent/AnswerProperties/TextAreaProperties.test.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { render, fireEvent, flushPromises } from "tests/utils/rtl";
+import TextAreaProperties from "./TextAreaProperties";
+import { colors } from "constants/theme";
+import { act } from "react-dom/test-utils";
+
+const renderTextProperties = (props) => {
+  return render(<TextAreaProperties {...props} />);
+};
```
Comment (MattElcock) [inline eq-author/src/components/AnswerContent/AnswerProperties/TextAreaProperties.test.js:9]:
```suggestion
const renderTextProperties = (props) => render(<TextAreaProperties {...props} />);
```

File: eq-author/src/components/AnswerContent/AnswerProperties/TextFieldProperties.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Required from "components/AnswerContent/Required";
+
+const TextFieldProperties = ({ answer, updateAnswer }) => {
+  return <Required answer={answer} updateAnswer={updateAnswer} />;
+};
```
Comment (MattElcock) [inline eq-author/src/components/AnswerContent/AnswerProperties/TextFieldProperties.js:7]:
```suggestion
const TextFieldProperties = ({ answer, updateAnswer }) => <Required answer={answer} updateAnswer={updateAnswer} />;
```

File: eq-author/src/components/ContentContainer/index.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        <Title className="content-title" data-test="content-title">
+          <HeaderText>{title}</HeaderText>
+        </Title>
+      </Header>
+      <Body className="collapsible-body" data-test="collapsible-body" isOpen>
+        {children}
+      </Body>
+    </Wrapper>
+  );
+};
```
Comment (MattElcock) [inline eq-author/src/components/ContentContainer/index.js:69]:
```suggestion
const ContentContainer = ({ title, children }) =>
    <Wrapper data-test="collapsible">
      <Header className="content-header" data-test="content-header">
        <Title className="content-title" data-test="content-title">
          <HeaderText>{title}</HeaderText>
        </Title>
      </Header>
      <Body className="collapsible-body" data-test="collapsible-body" isOpen>
        {children}
      </Body>
    </Wrapper>;
```

**Review by MattElcock — CHANGES_REQUESTED** [review]
I love this file. Fair play for making this.

`eq-author/src/components/AnswerContent/AnswerProperties/index.js`

Some really good refactoring and code changes in here. Nice one.

Just a few small style changes; functions that could be implicit returns, rogue comments etc.

**Review by MattElcock — APPROVED** [review]

**Review by ashtuck — APPROVED** [review]

**Pull request merged**